gradle/wrapper/gradle-wrapper.properties

```diff
@@ -16,4 +16,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 distributionUrl=https\://services.gradle.org/distributions/gradle-5.4.1-all.zip
 zipStoreBase=GRADLE_USER_HOME
-zipStorePath=wrapper/dists
+zipStorePath=wrapper/dists
```

Comment:
There's no need to modify this file, could you revert the change as well? Thanks.

Reply:
Thanks for checking this out. I have reverted these. Try to check again.